frontend/appflowy_flutter/lib/plugins/database_view/application/setting/group_bloc.dart

```diff
@@ -92,8 +106,10 @@ class DatabaseGroupEvent with _$DatabaseGroupEvent {
   const factory DatabaseGroupEvent.initial() = _Initial;
   const factory DatabaseGroupEvent.setGroupByField(
     String fieldId,
-    FieldType fieldType,
-  ) = _DatabaseGroupEvent;
+    FieldType fieldType, {
+    int? condition,
+    bool? hideEmpty,
+  }) = _DatabaseGroupEvent;
```

Comment:
You can use optional positional parameters here with `[...]`to maintain consistency

Reply:
Done.